tests/unit/mixins/ajax-request-test.js

```diff
@@ -879,6 +881,7 @@ describe('Unit | Mixin | ajax request', function() {
         .catch(function(reason) {
           expect(isTimeoutError(reason)).to.be.ok;
           expect(reason.payload).to.be.null;
+          expect(reason.status).to.equal(-1);
```

Comment:
Where did this `-1` come from?

Reply:
It comes from here: https://github.com/ember-cli/ember-ajax/pull/318/files#diff-6b893c2f1942ff5e1a6a907d293c60e3R83

Reply:
👍